web/static/js/content_types/image_tray.js

```diff
@@ -1,4 +1,7 @@
 import React from 'react'
+import ospryBuilder from '../utilities/ospry'
```

Comment:
@zberkom It's not super large, no. And it only gets included if Thesis is enabled and they can edit the current page.